python/hail/expr/functions.py

```diff
@@ -2738,10 +2739,18 @@ def max(*exprs) -> NumericExpression:
     single iterable expression (an array or set of numeric elements), or
     variable-length arguments of numeric expressions.
 
+    Note
+    ----
+    Missing arguments / array elements are ignored if `filter_na` is ``True``.
+    If `filter_na` is ``False``, then any missing element causes the result to
+    be missing.
+
     Parameters
     ----------
     exprs : :class:`.ArrayExpression` or :class:`.SetExpression` or varargs of :class:`.NumericExpression`
         Single numeric array or set, or multiple numeric values.
+    filter_na : :obj:`bool`
```

Comment:
kind of a style thing, but i like `filter_missing` since we otherwise refer to them as "missing elements"